codility_training/lessons.lesson10.PrimeAndCompositeNumbers.Flags/sangmandu.py

```diff
@@ -0,0 +1,36 @@
+def solution(A):
+    # write your code in Python 3.6
+    pass
+    B = []
+    for i in range(1, len(A) - 1):
+        if A[i - 1] < A[i] and A[i] > A[i + 1]:
+            B.append(i)
+
+    size = len(B)
+    if size == 0:
+        return 0
+
+    C = []
+    for i in range(1, size):
+        C.append(B[i] - B[i - 1])

```

Comment:
```suggestion
    C = [ B[i] - B[i - 1] for i in range(1, size) ]
```
마찬가지로 이렇게 표현할 수 있습니다!

Reply:
멋있습니다!!